.gitignore

```diff
@@ -9,5 +9,4 @@
 
 *.bundle
 *.so
-Gemfile.lock
```

Comment:
Unfrotunately, you have to keep it git ignored otherwise finding a bundler version that works on such a wide range of rubies is hard.

Reply:
I added `bundler: latest` in the CI config such that it'll use the latest compatible bundler of each Ruby in the matrix no matter what is being defined in the Gemfile.lock.